app/tasks/education_form/create_daily_spool_files.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+module EducationForm
+  class CreateDailySpoolFiles
```

Comment:
is this intended to be run on a schedule or just placed on the task queue to be picked up by a worker asynchronously?